demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -98,6 +97,10 @@ def __init__(
         if not id_set_path:
             id_set_path = 'Tests/id_set.json'
         self.id_set_path = id_set_path
+        # create the id_set only once per run.
+        self.id_set_validator = IDSetValidator(is_circle=self.is_circle, configuration=Configuration(),
```

Comment:
we have the `self.id_set_path` property in the validate_manager, i suggest using it when instantiating `IDSetValidator`, seems like the "./Tests/id_set.json" is hardcoded too many times.

Reply:
Not sure if I totally understand what needs to be done there, is the new commit what you meant?

Reply:
indeed. thx.